src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs

```diff
@@ -20,13 +20,20 @@ public class Ado : NotificationsBase, IAdo {
 
     public async Async.Task<OneFuzzResultVoid> NotifyAdo(AdoTemplate config, Container container, IReport reportable, bool isLastRetryAttempt, Guid notificationId) {
         var filename = reportable.FileName();
-        if (reportable is RegressionReport) {
-            _logTracer.LogInformation("ado integration does not support regression report. container:{Container} filename:{Filename}", container, filename);
-            return OneFuzzResultVoid.Ok;
+        Report? report;
+        if (reportable is RegressionReport regressionReport) {
+            if (regressionReport.CrashTestResult.CrashReport is not null) {
+                report = regressionReport.CrashTestResult.CrashReport;
+                _logTracer.LogInformation("parsing regression report for ado integration. report:{report}", report);
```

Comment:
have single log line ? so everything is better grouped in app insights ?

Reply:
Yup I actually meant to delete this.